### ISSUE TITLE
react-scripts Jest 27 dependency issue when using  astrouxds/react 

### ISSUE THREAD

**Issue by 321github123** [RocketCommunicationsInc/astro#132]
Hi,

The astrouxds/react dependency has ts-jest@27.0.7 as a dependency, which requires jest@^27.0.0 as a peer dependency. That is a problem because react-scripts doesn't seem to support jest@27. 

Somehow we'd like to force react-scripts to be okay with jest@^27.0.0 or for astrouxds/react to use an older version of ts-jest (26)

Would you have any suggestions on how to handle this problem?

Thank you.

**Comment by kit047** [comment]
Adding onto this, you can see the issue arise in the `react-starter` directory. Simply run `npm install`, and then run `npm list`, you see the peer dependency issue that has arisen. Here is the output from `npm list`:

```
npm ERR! peer dep missing: jest@^27.0.0, required by ts-jest@27.0.6
npm ERR! peer dep missing: typescript@>=3.8 <5.0, required by ts-jest@27.0.6
npm ERR! peer dep missing: @babel/core@^7.13.0, required by @babel/plugin-bugfix-v8-spread-parameters-in-optional-chaining@7.15.4
npm ERR! peer dep missing: typescript@>=2.8.0 || >= 3.2.0-dev || >= 3.3.0-dev || >= 3.4.0-dev || >= 3.5.0-dev || >= 3.6.0-dev || >= 3.6.0-beta || >= 3.7.0-dev || >= 3.7.0-beta, required by tsutils@3.21.0
npm ERR! peer dep missing: typescript@>=2.8.0 || >= 3.2.0-dev || >= 3.3.0-dev || >= 3.4.0-dev || >= 3.5.0-dev || >= 3.6.0-dev || >= 3.6.0-beta || >= 3.7.0-dev || >= 3.7.0-beta, required by tsutils@3.21.0
npm ERR! peer dep missing: typescript@>=2.8.0 || >= 3.2.0-dev || >= 3.3.0-dev || >= 3.4.0-dev || >= 3.5.0-dev || >= 3.6.0-dev || >= 3.6.0-beta || >= 3.7.0-dev || >= 3.7.0-beta, required by tsutils@3.21.0
```

**Comment by markacianfrani** [comment]
We'll remove that as a dependency in a hotfix ASAP, it should be a devDependency. In the meantime, you might be able to use npm i --force. But we'll have this fixed in the next day or two

**Comment by markacianfrani** [comment]
we just released 6.0.4 which should resolve this for you. feel free to reopen if you're still having issues
